packages/core/js/index.d.ts

```diff
@@ -0,0 +1,147 @@
+export const colorsWhite: string
+export const colorsBlack: string
+
+type ColorPalette10 = {
+  base: string
+  1: string
+  2: string
+  3: string
+  4: string
+  5: string
+  6: string
+  7: string
+  8: string
+  9: string
+  10: string
+}
+
+export const colorsBlue: ColorPalette10
+export const colorsTeal: ColorPalette10
+export const colorsGreen: ColorPalette10
+export const colorsLime: ColorPalette10
+export const colorsYellow: ColorPalette10
+export const colorsOrange: ColorPalette10
+export const colorsRed: ColorPalette10
+export const colorsPink: ColorPalette10
+export const colorsPurple: ColorPalette10
+export const colorsTextIcon: {
+  highOnDark: string
+  highOnLight: string
+  lowOnDark: string
+  lowOnLight: string
+}
+export const colorsBorder: {
+  highOnDark: string
+  highOnLight: string
+  lowOnDark: string
+  lowOnLight: string
+}
+export const colorsBackgroundDark: {
+  1: string
+  2: string
+  3: string
+}
+export const colorsBackgroundLight: {
+  1: string
+  2: string
+  3: string
+}
+export const colorsBackgroundUtility: {
+  base: string
+  25: string
+  30: string
+  40: string
+}
+export const colorsGradient: {
+  skillsBackground: string
+  skillsStop0: string
+  skillsStop100: string
+  flowBackground: string
+  flowStop0: string
+  flowStop100: string
+}
+export const colorsPrimaryAction: {
+  background: string
+  textDarkTheme: string
+  textLightTheme: string
+}
+export const colorsStatus: {
+  error: string
+  info: string
+  success: string
+  warning: string
+}
+export const colorsCode: {
+  white: string
+  gray: string
+  red: string
+  orange: string
+  yellow: string
+  green: string
+  turquoise: string
+  blue: string
+  purple: string
+  sand: string
+}
+
+export const accessibility: {
+  screenReaderOnly: {
+    position: string
```

Comment:
Feels like CSS props probably don't need listed out like this.  Just `screenReadOnly: object` ok?

Reply:
The `.d.ts` no longer applies if we go with the second commit, but I could explicitly add a type to `accessibility.ts` like this:

```typescript
const screenReaderOnly : object = {
  position: 'absolute',
  top: 'auto'
  [...]
}

export default {
  screenReaderOnly
}
```

Is your thinking that this is just an object that they'll be using for some css-in-js and may as well be opaque to them? That accessing the properties within it wouldn't be very helpful (and perhaps confusing)?

Reply:
Hey Justin, yeah, my comment isn't very applicable in light of your 2nd commit. I don't think this needs any additional treatment.